frontend/coprs_frontend/coprs/views/groups_ns/groups_general.py

```diff
@@ -41,12 +42,16 @@ def activate_group(fas_group):
         db.session.add(group)
         db.session.commit()
 
-        flask.flash(
-            "FAS group {} is activated in the Copr under the alias {} "
-            .format(fas_group, alias)
-        )
+        # can't have more than one alias for a group
+        if group.name != alias:
+            message = "Group {} already activated as alias {} ".format(
+                fas_group, group.name)
+        else:
+            message = "Group {} is activated in the System under the alias {} ".format(
+                fas_group, alias)
```

Comment:
:-) how do you trigger this one actually?  Is there a reproducer?

Reply:
> :-) how do you trigger this one actually? Is there a reproducer?

Yeah, i just go `/groups/activate/<group_name>` twice with two different alias
The second submit will lead to a 404 page

Reply:
AAAh, this. Yeah, I totally misunderstood the thing you want to fix (I though you want to map 2 external groups to a single Copr group, and it is vice versa).

Reply:
Could you please return "400 Bad Request" if user visits `/groups/activate/<existing_group>` for the second time?  There's no such link, so that must be an error anyway (processing the request seems just redundant).

Reply:
Will simplely return 400 be a good experience?
Or just redirect user to the `existed group`?

Reply:
Not sure.  Do you think users can get to the route once it is activated?   If yes, redirecting is indeed a better option.

Reply:
> Not sure. Do you think users can get to the route once it is activated? If yes, redirecting is indeed a better option.

Seems no possible unless they know the route and play with it like me :)
But return 400 or redirect will not waste anything, so why not?